site/content/in-dev/unreleased/metastores.md

```diff
@@ -32,6 +32,29 @@ In order to add other JDBC drivers, you have to build Polaris using the `eclipse
 
 ## Polaris Server Configuration
 
+#### Relational JDBC
+Configure the `polaris.persistence` section in your Polaris configuration file
+(`application.properties`) as follows:
+
+```
+polaris.persistence.type=relational-jdbc
+
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.username=<your username>
+quarkus.datasource.password=<your password>
+
+quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/<database_name>
+```
+
+The relational JDBC metastore is a Quarkus-managed datasource and only supports Postgres and H2 as of now.
+Please refer to the documentation here:
+[Configure data sources in Quarkus](https://quarkus.io/guides/datasource)
+
+Note: Polaris will always create schema 'polaris_schema' during bootstrap under the configured database.
+
```

Comment:
Nit: we say that the relational JDBC persistence supports Postgres and H2 (just like EclipseLink), but we only document how to use it with Postgres.  That could suggest that either H2 is not usable with that persistence implementation or could mislead people into using EclipseLink.  Should there be a `#### Using Postgres` and a `#### Using H2` subsection?

Reply:
Can we also say more about _why_ it's limited to specific RDBMSs?

Reply:
AFAIK, the focus on H2 is to allow local and in-memory operation. I'm not sure it is worth documenting it in the "docs" files for end users :thinking: I'd say README notes about H2 should be sufficient.

Reply:
It should be possible to support other RDBMS (e.g. MariaDB and MySQL by extension) will small additional effort, I think.

Reply:
yes, MariaDB and MySQL should be able to supported, easily this effort mostly added PGSQL, we don't use any PGSQL sepecific feature during query construction, we just need the bootstrap script and the error code handling,


------ 

Sure thing let me document that !